hirzenbach-bot/commands.py

```diff
@@ -114,21 +102,24 @@ async def inspire(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
 
 
 async def generic_message(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
-    data = Data.read()
-    if not update.effective_chat.id in data.memory:
-        data.memory[update.effective_chat.id] = list()
-    chat_memory = _append_to_memory(
+    _append_to_memory(
         update.effective_chat.id,
         update.effective_user.first_name,
         update.effective_message.text,
     )
+    chat_memory = persistence.get_memory(update.effective_chat.id)
 
     reply_to_message = update.effective_message.reply_to_message
     is_reply_to_me = (
         reply_to_message is not None and reply_to_message.from_user.id == context.bot.id
     )
     is_addressing_me = context.bot.name in update.effective_message.text
 
+    fraction_match = _FRACTION_REGEX.match(update.effective_message.text)
```

Comment:
You can register a handler directly for this regex in `main.py`

Reply:
Uu that's really cool! [fc2a085](https://github.com/janhohenheim/hirzenbach-bot/pull/1/commits/fc2a085ca1ff15676d9e7502e4c5f6355a930765)